docs/guides/how_to_guides/creating_and_editing_expectations/spare_parts/how_to_create_expectations.rst

```diff
@@ -1,4 +1,4 @@
-.. _tutorial_create_expectations:
+.. _tutorials__create_expectations:
 
 How to create Expectations
 ==========================
```

Comment:
Warning below can now be removed!

Reply:
Also, please capitalize "Expectations" per new style guide in the steps to be looped over (again, realize this isn't from you originally, but while we're editing this tutorial).